### DISCUSSION TITLE
Add Portuguese Translation for documentation website

### DISCUSSION THREAD

**Discussion by lucas-goldner** [rrousselGit/riverpod#2843] [General]
Portuguese is a language that still needs to be added to the website, and I have not seen an issue on that.
Does it make sense to start working on that or would it make sense to wait since the rework of the documentation is going on?

**Comment by rrousselGit** [comment]
That's up to you. You can start working on it now, or wait for the rework

**Comment by catalunha** [comment]
I am Brazilian. We have huge Flutter communities here. Riverpod is my favorite state and I would love to master English to the point of taking on this valuable mission. But I'm sorry because my English is only technical and sufficient for my consumption.
